lexicons/tools/ozone/moderator/addUser.json

```diff
@@ -0,0 +1,41 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.moderator.addUser",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Add a user as moderator in the ozone service. Requires admin role.",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["did", "role"],
+          "properties": {
+            "did": { "type": "string", "format": "did" },
+            "role": {
+              "type": "string",
+              "knownValues": [
+                "tools.ozone.moderator.defs#modRoleAdmin",
+                "tools.ozone.moderator.defs#modRoleModerator",
+                "tools.ozone.moderator.defs#modRoleTriage"
+              ]
+            }
+          }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "ref",
+          "ref": "tools.ozone.moderator.defs#user"
+        }
+      },
```

Comment:
we discussed most of these outside of github but leaving some traces of that here.
I kinda like the entity specific namespace since it isolates the ACL management in one place. `tools.ozone.admin` will probably end up with a lot more than ACL management, eventually.
also, in my mind, all users in ozone are moderators so, the fact that we have a role named `moderator` is the only confusing part. could we rename that to something like `escalation`? I like the term moderator instead of user because it simplifies the definition of ozone users in the sense that any `user` in the network can be a `moderator`. 

with that, my counter proposal to clarify and simplify the terminologies:
- the namespace remains `tools.ozone.moderator` and we ditch the term user everywhere so, `tools.ozone.moderator.addUser` becomes `tools.ozone.moderator.add`
- rename the `modRoleModerator` to `modRoleEscalation` (probably needs a better name)

Reply:
We use `actor` in a couple other Lexicons, seems like it would be a good fit here? `tools.ozone.actor.*`. I also like "Member" as in "team member"... that also points to `tools.ozone.team.*`.

In all these terms we need to differentiate between a general atproto "user" / "account" from somebody who can log in to the Ozone instance specifically.

To propose something specific:

* use the namespace `tools.ozone.team.*`
* use the team "member" for any account which is part of the team
* keep the existing triage / moderator / admin terminology for now.